tests/conftest.py

```diff
@@ -51,6 +51,18 @@ def owned_manifest():
         return json.load(file_obj)
 
 
+@pytest.fixture
+def piper_coin_manifest():
+    with open(str(V2_PACKAGES_DIR / "piper-coin" / "1.0.0-pretty.json")) as file_obj:
+        return json.load(file_obj)
```

Comment:
Not a big deal, but an optimization you could do for these.

```
@pytest.fixture(scope="session")
def _piper_coin_manifest():
    with open(str(V2_PACKAGES_DIR / "piper-coin" / "1.0.0-pretty.json")) as file_obj:
        return json.load(file_obj)

@pytest.fixture
def piper_coin_manifest(_piper_coin_manifest):
    return copy.deepcopy(_piper_coin_manifest)
```

This way we only every hit the disk once per test run.  Alternatively, you could just load the manifest as a top level module variable instead of the `_piper_coin_manifest` fixture, and just return a copy of that variable.